app/controllers/application_controller.rb

```diff
@@ -40,15 +40,25 @@ def configure_permitted_parameters
     end
   end
 
+  # Check user's password against pwned password service and warn if necessary
+  def check_for_pwnage( resource )
+    return unless resource.respond_to?( :pwned? ) && resource.pwned?
```

Comment:
I can't remember how the `pwned?` check works in the test environment. 

It could be making a HTTP request to a remote API every time a test logs a user in, slowing the tests down or possibly breaking them if the haveibeenpwned API is down. 

Remote API calls are usually best avoided during tests for these reasons.

We can workaround this a few ways I expect. Let me know if its making remote API calls.

Reply:
https://github.com/denny/ShinyCMS-ruby/blob/master/app/models/user.rb#L8 skips this during tests.